### PR TITLE
Scraper test

### PR DESCRIPTION
**Pull request by ChristianHolm1**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -161,3 +161,4 @@ cython_debug/
 
 # Other
 .vscode
+*.png
```
Comment (OHMelin) [inline .gitignore:164]: Do we still need this?

File: price_match/admin.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
 from django.contrib import admin
+from price_match.models import Product
+from price_match.models import Config
```
Comment (OHMelin) [inline price_match/admin.py:3]: Please combine these two, so we can have them as one line

File: price_match/models.py [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+    accepted = models.BooleanField(default=False)
+    creation_datetime = models.DateTimeField(auto_now_add=True)
+    acceptance_datetime = models.DateTimeField(null=True, blank=True)
+    product_image = models.ImageField(null=True, blank=True)
+
+    def save(self, *args, **kwargs):
+        self.total_price = self.price + self.shipping_price
+        super(Product, self).save(*args, **kwargs)
+
+    def __str__(self):
```
Comment (OHMelin) [inline price_match/models.py:21]: Do we still need this str method?

File: price_match/models.py [hunk 4]
```diff
@@ -1,3 +1,49 @@
 from django.db import models
 
-# Create your models here.
+class Product(models.Model):
```
Comment (OHMelin) [inline price_match/models.py:3]: Should we define a primary key?

File: price_match/utils.py [hunk 5]
```diff
@@ -0,0 +1,122 @@
+from datetime import datetime, timedelta
```
Comment (OHMelin) [inline price_match/utils.py:1]: Move this two down as it starts with "from"

File: price_match/utils.py [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+from price_match.models import Product, Config
+
+
+def scrape_website(url: str) -> Product:
+    product = Product()
+    product.url = url
+    
+    if not __product_allready_accepted(product.url):
+
```
Comment (OHMelin) [inline price_match/utils.py:21]: Remove this empty line

**Review by OHMelin — CHANGES_REQUESTED** [review]
Overall good. No major issues

File: price_match/models.py (hunk 3 above)
Comment (MagNsch) [reply price_match/models.py:21]: Yes

**Review by OHMelin — APPROVED** [review]
Nice job!

**Review by MagNsch — APPROVED** [review]

**Pull request merged**